src/Integration/Connection/ConnectionWorkflow.cs

```diff
@@ -55,7 +55,7 @@ public ConnectionWorkflow(IHost host, ICommand parentCommand)
             this.host = host ?? throw new ArgumentNullException(nameof(host));
             this.parentCommand = parentCommand ?? throw new ArgumentNullException(nameof(parentCommand));
 
-            this.credentialStore = this.host.GetService<ICredentialStoreService>();
```

Comment:
`ICredentialStoreService` was already MEF-creatable, and was MEF-created by most consumers. However, it was still exposed as an `ILocalService`.

The ConnectionWorkflow was the only consumer that still fetched it using the service provider mechanism. The consumer is easy to fix up - just request the service as a MEF service instead.